resources/public/cheatSheet.html

```diff
@@ -0,0 +1,65 @@
%1FeDE-605%1E047A/03+%1FrDE-605%1E047C+%1FSpnd%1Fia%1FaPechmann%2C+Wilhelm+/von%1F0174129386%1E050C+%1Famaschinell+eingespielte+Altdaten%1E050G+%1Fb1.2%3A+%22Ex+libris+Wilhelmi+Liberi+Baronis+de+Pechmann%22%1E%0A001A+%1F09013%3A26-09-11%1E001B+%1F09999%3A31-07-13%1Ft01%3A16%3A49.000%1E001D+%1F09013%3A26-09-11%1E001U+%1F0utf8%1E001X+%1F00%1E002@+%1F0Tp6%1E003@+%1F0174168926%1E007K+%1Fagnd%1F0174168926%1E008@+%1Fazu%1E039I+%1F9174039441%1F7Tp3%1FVpiz%1FAgnd%1F0174039441%1FaMolisch%1FdHans%1E%0A001A+%1F00012%3A30-04-97%1E001B+%1F09999%3A05-04-08%1Ft22%3A04%3A47.000%1E001D+%1F09999%3A05-04-08%1E001U+%1F0utf8%1E001X+%1F00%1E002@+%1F0Tp1%1E003@+%1F0104379642%1E003U+%1Fahttp%3A//d-nb.info/gnd/104379642%1E004B+%1Fapiz%1E007K+%1Fagnd%1F0104379642%1E007N+%1Fapnd%1F0104379642%1Fvzg%1E008A+%1Faa%1Faf%1E028A+%1FdAndre%CC%81%1FaBaiole%1E042B+%1FaXA-FR%1E047A/03+%1FeDE-12%1E047A/03+%1FrDE-12%1E047C+%1FSpnd%1Fia%1FaBaiole%2C+Andre%CC%81%1F0104379642%1E050C+%1FaHBZ%1E050E+%1FaBN%1E050G+%1FbFranz.+Theologe%1E060R+%1Fcum+1649%1F4datl%1E%0A001A+%1F00103%3A31-07-96%1E001B+%1F07777%3A30-07-13%1Ft11%3A05%3A32.000%1E001D+%1F09999%3A05-04-08%1E001U+%1F0utf8%1E001X+%1F00%1E002@+%1F0Tp1%1E003@+%1F0117305553%1E003U+%1Fahttp%3A//d-nb.info/gnd/117305553%1E004B+%1Fapiz%1E007K+%1Fagnd%1F0117305553%1E007N+%1Fapnd%1F0117305553%1Fvzg%1E008A+%1Faz%1E008B+%1Fak%1E028@+%1FdL...%1FaStrackerjan%1E028A+%1FdLudwig%1FaStrackerjan%1E032T+%1Fam%1E041R+%1F9040290506%1F7Ts1%1FVsaz%1FAgnd%1F04029050-5%1FaJurist%1F4berc%1E041R+%1F9040533093%1F7Ts1%1FVsaz%1FAgnd%1F04053309-8%1FaSchriftsteller%1F4beru%1E042B+%1FaXA-DE%1E047A/03+%1FeDE-611%1E047A/03+%1FrDE-611%1E047C+%1FSpnd%1Fia%1FaStrackerjan%2C+Ludwig%1F0117305553%1E050C+%1FaNDSBIO%1E060R+%1Fa1825%1Fb1881%1F4datl%1E%0A001A+%1F00255%3A08-01-02%1E001B+%1F09999%3A02-01-12%1Ft17%3A16%3A36.000%1E001D+%1F09999%3A06-04-08%1E001U+%1F0utf8%1E001X+%1F00%1E002@+%1F0Tp1%1E003@+%1F0123464595%1E003U+%1Fahttp%3A//d-nb.info/gnd/123464595%1E004B+%1Fapiz%1E007K+%1Fagnd%1F0123464595%1E007N+%1Fapnd%1F0123464595%1Fvzg%1E007N+%1Faswd%1F04666643-6%1Fvzg%1E008A+%1Fas%1E028A+%1FdSiliano%1FaSimoncini%1E042A+%1Fa13.4p%1E042B+%1FaXA-IT%1E047A/03+%1FeDE-255%1E047A/03+%1FrDE-255%1E047C+%1FSpnd%1Fia%1FaSimoncini%2C+Siliano%1F0123464595%1E050E+%1FaInternet%1E050G+%1FbItal.+Maler%1E060R+%1Fakein-jahr%1F4datl%1FvVorlage%1E%0A001A+%1F09999%3A23-01-02%1E001B+%1F09999%3A05-04-08%1Ft20%3A38%3A46.000%1E001D+%1F09999%3A05-04-08%1E001U+%1F0utf8%1E001X+%1F00%1E002@+%1F0Tp6%1E003@+%1F0126744327%1E003U+%1Fahttp%3A//d-nb.info/gnd/126744327%1E004B+%1Fapiz%1E007K+%1Fagnd%1F0126744327%1E007N+%1Fapnd%1F0126744327%1Fvzg%1E008A+%1Faf%1E008B+%1Fav%1E028A+%1FdHermann%1FaMa%CC%88dler%1E046G+%1FaMehr+fu%CC%88r+Hamburg+-+weniger+fu%CC%88r+die+Ru%CC%88stung%1E047A/03+%1FeDE-101%1E047A/03+%1FrDE-101%1E047C+%1FSpnd%1Fia%1FaMa%CC%88dler%2C+Hermann%1F0126744327%1E050C+%1FaMaschinell+erzeugt+aus+DBL-Retro-Daten%3BMaschinell+verknuepft+mit+DBL-Retro-Titeldaten%1E060R+%1Fc1970%1F4datw%1E&active-editor=fix">
```

Comment:
There is already a usecase example: https://metafacture.org/playground/?example=Count_patterns_with_triples